specification/keyvault/resource-manager/readme.md

```diff
@@ -209,7 +209,6 @@ This is not used by Autorest itself.
 
 ``` yaml $(swagger-to-sdk)
 swagger-to-sdk:
-  - repo: azure-sdk-for-net
```

Comment:
@ShivangiReja can you confirm whether mgmt plane track 2 is also supported with the ps script (eng/scripts/Automation-Sdk-Generate.ps1) you wrote for `azure-sdk-for-net-track2`?

@heaths as mgmt plane track2 SDKs require an `autorest.md` file in `azure-sdk-for-net` repo and a default one won't work for most RPs based on our experience (many RPs do not pass the more strict swagger correctness requirement of .net track 2), we may only add `azure-sdk-for-net-track2` for SDKs that have released beta packages and keep adding it to more RPs when we support more track2 packages.

Reply:
If track 2 is not ready, why are we removing this line now? I am guessing we will keep releasing track 1 till track 2 SDKs for our repo is GA right? won't this break our track 1 SDK generation? I didn't see any explanation on why these lines are being remove now in the PR.

Reply:
@fengzhou-msft Yes, [Automation-Sdk-Generate.ps1](https://github.com/Azure/azure-sdk-for-net/blob/main/eng/scripts/Automation-Sdk-Generate.ps1) script supports SDK automation for both dataplane and management plane track-2 SDKs.

Reply:
> If track 2 is not ready, why are we removing this line now? I am guessing we will keep releasing track 1 till track 2 SDKs for our repo is GA right? won't this break our track 1 SDK generation? I didn't see any explanation on why these lines are being remove now in the PR.

.net track1 is out of maintenance for a while and so is the pipeline script for generating it. For new services, we no longer support onboarding track1 sdks. For existing services, the track1 SDK generation step fails a lot for reasons like it will compile test code as well which needs to be fixed manually with breaking changes. It becomes a noise during swagger review. On the other hand, track1 SDKs can still be generated and updated with the `generate.ps1` script in `azure-sdk-for-net` repo until we retire track1 officially. We are only removing the autogen step in swagger PRs, which has become more buggy than helpful.

Reply:
This isn't just about mgmt plane SDKs, though. Data plane SDKs also use code gen. In fact, both sdk/keyvault/Azure.ResourceManager.KeyVault and sdk/keyvault/Azure.Security.KeyVault.Administration contain autorest.md files and use code gen. So just removing the track 1 pipeline now means .NET track 2 SDKs aren't being built for Key Vault, which is a regression in coverage.

Reply:
Let me open another PR to first remove track1 and add track2 for the RPs that are support by track2 to avoid the regression in coverage, @heaths.

Reply:
@fengzhou-msft As .net track1 is removed and there is only `azure-sdk-for-net-track2`, cloud you rename `azure-sdk-for-net-track2` to `azure-sdk-for-net`, which means `azure-sdk-for-net` generates codes for mgmt track2 sdk. You can find java/js sdk only have one configuration, such as `azure-sdk-for-java`, or `azure-sdk-for-js`, which represents track2 sdk.

Reply:
Does #18996 supersede this one, then? Seems like it should since it's also removing track 1 validation.

Reply:
@dw511214992 how about we do that after we have a high enough coverage of the track 2 SDKs as @heaths mentioned about the concern of a regression in coverage. It would likely to be the time we are about to announce the deprecation of track 1.